### PR TITLE
refactor: post comments

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/core/directives/dropdown/dropdown-context.directive.ts [hunk 1]
```diff
@@ -34,9 +34,9 @@ const MOVE_THRESHOLD = 15;
             'onTouchMove($event.touches[0].clientX, $event.touches[0].clientY)',
         '(touchstart.silent.passive)':
             'onTouchStart($event.touches[0].clientX, $event.touches[0].clientY)',
-        '(document:pointerdown.silent)': 'closeDropdown($event.target)',
-        '(document:contextmenu.capture.silent)': 'closeDropdown($event.target)',
-        '(document:keydown.esc)': 'closeDropdown($event.target)',
+        '(document:pointerdown.silent)': 'closeDropdown($event.currentTarget)',
```
Comment (waterplea) [inline projects/core/directives/dropdown/dropdown-context.directive.ts:37]: It was:
```ts
    @HostListener('document:pointerdown.silent', ['$event.target'])
    @HostListener('document:contextmenu.capture.silent', ['$event.target'])
    @HostListener('document:keydown.esc', ['$event.currentTarget'])
```
So `currentTarget` only on the last one.

File: projects/addon-doc/index.ts [hunk 2]
```diff
@@ -29,7 +28,6 @@ export const TuiAddonDoc = [
     TuiDocPageTabConnector,
     TuiDocNavigation,
     TuiDocMain,
-    TuiDocText,
```
Comment (waterplea) [inline projects/addon-doc/index.ts:32]: Maybe we should deprecate and keep it? In case somebody was using it

**Review by waterplea — APPROVED** [review]

File: projects/core/directives/dropdown/dropdown-context.directive.ts (hunk 1 above)
Comment (splincode) [reply projects/core/directives/dropdown/dropdown-context.directive.ts]: fixed

File: projects/addon-doc/index.ts (hunk 2 above)
Comment (splincode) [reply projects/addon-doc/index.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**